templates/supplier_app/invoices_form.html

```diff
@@ -94,6 +94,15 @@ <h4 class="text-center">New Invoice</h4>
                                 {% endif %}
                             </div>
                         </div>
+                        <div class="row">
+                            <div class="form-group col-md-2">
```

Comment:
if you want to add another form-group on that column, this html structure will support it?